app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt

```diff
@@ -42,8 +43,11 @@ class EditActivity : AppCompatActivity() {
         supportActionBar?.title = ""
         val intent = intent
         imageUri = intent.getStringExtra("image") ?: ""
-        vm = ViewModelProvider(this).get(EditViewModel::class.java)
+        vm = ViewModelProvider(this)[EditViewModel::class.java]
         val sourceExif = imageUri.toUri().path?.let { ExifInterface(it) }
+        // TODO(Deprecation : 'TAG_APERTURE: String' is deprecated. Deprecated in Java)
```

Comment:
For Maintainers : should we create separate issue for this referencing deprecation for it and comment issue here?

Reply:
Yes thanks! 🙂